_posts/2024-01-11-gsoc2023_xhgchen.md

```diff
@@ -0,0 +1,161 @@
+
+[PR #29](https://github.com/MDAnalysis/transport-analysis/pull/29), which adds plotting functionalities for the Einstein-Helfand viscosity class, is essentially complete but has yet to be merged because we are still considering whether it would be better to leave the plot in standard MDAnalysis units or convert them to SI units. While standard MDAnalysis units are friendly for VACFs and self-diffusivity, that is not the case for viscosity due to the number of terms involved in the calculation. It is the only unmerged PR in the project at the time of writing.
+
+Because we prioritized the new, more user-friendly Einstein-Helfand viscosity class, which turned out to be more time consuming due to the research and complex implementation, I did not get to write the Green-Kubo viscosity utility function within the GSoC timeline. Nonetheless, I am happy to have adapted my original plan to our community discussions and delivered a more useful project. I plan to implement the utility function after GSoC, though I suspect most users will much prefer the Einstein-Helfand viscosity class. Ultimately, I achieved my goals for this summer and I am excited to see Transport Analysis grow as a community resource.
+
+### Reproduction of the Literature
+
+To go beyond testing against simple "toy" systems and test data, I began a reproduction of established transport property calculations in the literature to validate my implementations. I successfully calculated a self-diffusivity of $2.47 \times 10^-9$ $m^2 / s$ using my Green-Kubo self-diffusivity implementation from a $20$ $ps$ simulation of the SPC/E water model at $298$ $K$, which agrees well with the results in the literature.[^3]<sup>,</sup> [^4] A full writeup of this reproduction can be found in my last blog post, [*Reproduction and Beyond GSoC (GSoC '23, 4)*](https://xhgchen.github.io/posts/reproduction-and-beyond-gsoc/). Some of the key plots are provided below.
+
+#### VACF Plot
+
+![VACF Plot](/assets/img/2023-08-22/vacf_plot.PNG){: width="1280" height="960" }
+
+#### Running Integral Plot
+
+![Running Integral Plot](/assets/img/2023-08-22/running_integral_plot.PNG){: width="1280" height="960" }
+
+## Using Transport Analysis
+
+### Installation
+
+Transport Analysis is nearing release, and it can be cloned and installed from GitHub in a virtual environment. Here is a simple example using `mamba`.
+
+First, we clone Transport Analysis:
+
+```console
+git clone https://github.com/MDAnalysis/transport-analysis.git
+```
+
+Then we can go ahead and install the package with `mamba` and `pip`. In this case, I have named the environment `ta-package`:
+
+```console
+cd transport-analysis
+
+mamba create -n ta-package
+
+mamba env update --name ta-package --file devtools/conda-envs/test_env.yaml
+
+mamba activate ta-package
+
+pip install -e .

```

Comment:
I'd remove blank lines to make the code display more compact.

Reply:
Done.